utils/mariadb/Makefile

```diff
@@ -361,8 +362,10 @@ CMAKE_OPTIONS += \
 	-DINSTALL_SQLBENCHDIR="" \
 	-DINSTALL_SUPPORTFILESDIR=share/mariadb \
 	-DINSTALL_UNIX_ADDRDIR=$(MARIADB_SOCKET) \
+	-DHOSTNAME="uci get system.@system[0].hostname" \
 	-DMYSQL_DATADIR=/srv/mysql \
 	-DMYSQL_UNIX_ADDR=$(MARIADB_SOCKET) \
+	-DSEMODULE_PACKAGE_EXECUTABLE=NOT-FOUND \
 	-DSKIP_TESTS=ON \
 	-DWITH_DEBUG=OFF \
```

Comment:
I believe that `WITH_DEBUG` is something that the `CMakeLists.txt` of MySQL cares about, but has no effect with MariaDB. MariaDB follows the CMake way of building things: `CMAKE_BUILD_TYPE=RelWithDebInfo` or `CMAKE_BUILD_TYPE=Release` is what we would want.

There are a few `CMAKE_OPTIONS +=` assignments before this one. Maybe they could be consolidated? I believe that `-DSTACK_DIRECTION=-1` was made redundant by MariaDB/server@d0abbdf56e11ccc88447c1dc80caaf355c94be3b.